README.md

```diff
@@ -45,6 +45,7 @@ Conference | Dates | Proposals | Location | Twitter
 [HolyJS](https://holyjs-moscow.ru/en/)* | Nov 24th - 25th | [Sep 24th](https://holyjs-moscow.ru/en/callforpapers/) | Moscow, Russia | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/HolyJSconf)
 [DevFest Siberia](https://gdg-siberia.com) | Nov 23rd - 25th | [Jul 31st](https://cfp.exchange/cfp/details/devfest-siberia-2018) | Novosibirsk, Russia
 [NodeFest](https://nodefest.jp/2018/) | Nov 23th - 24th | [Sep 30th](https://docs.google.com/forms/d/1b3DFCCVIK1rinVMFO39q6hSUev1Q_27avmYRE2E4YG0/edit) | Tokyo, Japan | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/nodefest)
+[JSDC-TW](http://2018.jsdc.tw/) | Nov 17th |  | Taipei, Taiwan | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/jsdc_tw)
```

Comment:
Thanks for updating the PR 

I'm sorry if I was unclear in my previous comment about the `unknown` link, what I ment was to replace `[unknown](#)` with just the string `unknown` in the 3rd column. I can also just do that my self to not bother you with it 😃